modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc

```diff
@@ -30,7 +30,21 @@ $ rosa create account-roles --hosted-cp --mode auto --yes
 +
 [source,terminal]
 ----
-$ export ACCOUNT_ROLES_PREFIX="${ACCOUNT_ROLES_PREFIX}"
+$ export ACCOUNT_ROLES_PREFIX=<account_role_prefix>
+----
+
+** View the value of the variable by running the following command:
++
+[source,terminal]
+----
+$ echo $ACCOUNT_ROLES_PREFIX
+----
++
+.Sample output
++
+[source,terminal]
+----
+$ ManagedOpenShift
```

Comment:
I don't think you need the `$` for return values. I have also seen people use "source,text" for the example box. Just an idea.